NativeCore/ReClassNET_Plugin.hpp

```diff
@@ -1,4 +1,6 @@
 #pragma once
+#ifndef __RECLASS_NET_PLUGIN_H__
+#define __RECLASS_NET_PLUGIN_H__
```

Comment:
Why is this needed? GCC understands `#pragma once`

Reply:
its good practice in my opinion since it was windows specific for quite some time